### ISSUE TITLE
Bank tag issue

### ISSUE THREAD

**Issue by scape4jesus** [runelite/runelite#6577]
Some of my bank tags are not working/partially working. My "vorkath" tag seems to bring up all of the relevant gear, but when I type "zulrah", it only brings up 4 items, similar thing happens when I type "gorillas" or "raids". I have checked the other items and they have the tag assigned to them. I tried deleting all tags for an item, then re-adding them, didn't work. I updated windows and rebooted my computer, disabled/enabled plugin, disabled, closed client, re-launched, enabled again and a bunch of combinations of those sorts of things. Nothing seems to be working.

**Comment by deathbeam** [comment]
Are you sure you are separating tags with commas and not spaces? That is common mistake people do

**Comment by scape4jesus** [comment]
Yeah. I didn't even know that was a thing. I never had an issue with spaces before>.< Appears to have fixed it. I will post again if it pops up again in some way.
